src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -1117,9 +1114,11 @@ void runtime::cpu::CPU_ExternalFunction::build()
     {
         return;
     }
-
+    // stream writer to dump the debug manifest for the DEX
+    static const string s_debug_dir = "debug";
```

Comment:
It would be ideal to dump the generated code and dex manifests in the same directory. Lets keep it as cpu_codegen for now.